src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -589,6 +635,57 @@ void FindEnzymesDialog::accept() {
     QDialog::accept();
 }
 
+void FindEnzymesDialog::sl_setEnzymesBySuppliers(QStringList checkedSuppliers) {
```

Comment:
nit: you do more than update enzymes in this method. It is better to rename it to something like `sl_handleSupplierSelectionChange`

Reply:
OK